projects/api-client/src/api/branch.service.ts

```diff
@@ -2,7 +2,7 @@
  * Wharf main API
  * Wharf backend API that manages data storage for projects, builds, providers, etc.
  *
- * OpenAPI spec version: v4.1.0
+ * OpenAPI spec version: local docker-compose
```

Comment:
Do we even need this comment? Seems like not having it in the generate template at all would be better for our flow.

Reply:
I like it in the generated code. At least if we had it in one place. Having them in all files is perhaps overkill.